webcash/walletclient.py

```diff
@@ -128,8 +128,10 @@ def load_webcash_wallet(filename=WALLET_NAME):
 
 # TODO: encryption
 def save_webcash_wallet(webcash_wallet, filename=WALLET_NAME):
-    with open(filename, "w") as fd:
+    temporary_filename = f"{filename}.{os.getpid()}"
+    with open(temporary_filename, "w") as fd:
         fd.write(json.dumps(webcash_wallet))
+    os.rename(temporary_filename, filename)
```

Comment:
The python docs for `os.rename` says that if the destination file exists, then it will fail with an `OSError`:
https://docs.python.org/3/library/os.html#os.rename

Reply:
`os.replace` instead should work.